src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/ManagedIndexRunner.kt

```diff
@@ -127,6 +133,9 @@ object ManagedIndexRunner : ScheduledJobRunner,
     private val errorNotificationRetryPolicy = BackoffPolicy.exponentialBackoff(TimeValue.timeValueMillis(250), 3)
     private var jobInterval: Int = DEFAULT_JOB_INTERVAL
     private var allowList: List<String> = ALLOW_LIST_NONE
+    // whether old metadata in cluster state is delete successful last time
+    private var metadataDeleted: Boolean = true
```

Comment:
The runner is a singleton, you'll have a bunch of jobs all updating this value together. Is that intention?

Reply:
removed